mkdocs.yml

```diff
@@ -19,11 +19,14 @@ nav:
       - 'Building flashing and debugging': building.md
       - 'Running': running.md
       - 'TPM driver instrumentation': instrumentation.md
+      - 'RaspberryPi demonstration with TPM module':
+        - 'Connecting Raspberry Pi with Windows OS': rpi_connection.md
+        - 'Running TPM 2.0 module on Raspberry Pi': rpi.md
+        - 'FAQ': rpi_faq.md
     - 'Target TPM hardware interfaces': hardware_interfaces.md
+    - 'Mechanical design':
+      - 'TPM modules pinout': tpm_modules_pinout.md
+      - 'TPM modules renders': tpm_renders.md
     - 'Current issues': issues.md
-    - 'TPM modules pinout': tpm_modules_pinout.md
     - 'Journal': journal.md
-    - 'RaspberryPi demonstration with TPM module':
-      - 'Connecting Raspberry Pi with Windows OS': rpi_connection.md
-      - 'Running TPM 2.0 module on Raspberry Pi': rpi.md
-      - 'FAQ': rpi_faq.md
+    
```

Comment:
remove empty line

Reply:
Empty line removed in commit `d6c9c0644891819aae6b6107230aead4950a9d20`